dev-util/idea/idea-15_pre.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+MY_PV="142.3926.4"
+MY_PN="${PN}-IU"
+
+DESCRIPTION="A complete toolset for web, mobile and enterprise development."
+HOMEPAGE="http://www.jetbrains.com/idea"
+SRC_URI="https://download.jetbrain.com/idea/ideaIU-${MY_PV}.tar.gz"
+
+LICENSE="IDEA IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal"
+SLOT="0"
+IUSE=""
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=virtual/jdk-1.7"
+S=${WORKDIR}/${MY_PN}-${MY_PV}
+
+src_install() {
+	local dir="/opt/${PN}"
+
+	insinto "${dir}"
+	doins -r *
+	fperms 755 ${dir}/bin/{idea.sh,fsnotifier{,64}}
+
+	# Create /usr/bin/intelliJ
+	dodir /usr/bin/
+	cat <<-EOF >"${ED}"usr/bin/${PN}
```

Comment:
I don't think `${ED}` is guaranteed to end with a `/`. Afaik, the best workaround is to do `"${ED%/}"/usr/bin/${PN}`